clients/java/src/main/java/io/camunda/zeebe/client/api/command/CommandWithOneOrMoreTenantsStep.java

```diff
@@ -35,52 +34,39 @@ public interface CommandWithOneOrMoreTenantsStep<T> extends CommandWithTenantSte
    * @since 8.3
    */
   @Override
-  @ExperimentalApi("https://github.com/camunda/zeebe/issues/12653")
   T tenantId(String tenantId);
 
   /**
-   * <strong>Experimental: This method is under development, and as such using it may have no effect
-   * on the command builder when called. While unimplemented, it simply returns the command builder
-   * instance unchanged. This method already exists for software that is building support for
-   * multi-tenancy, and already wants to use this API during its development. As support for
-   * multi-tenancy is added to Zeebe, each of the commands that implement this method may start to
-   * take effect. Until this warning is removed, anything described below may not yet have taken
-   * effect, and the interface and its description are subject to change.</strong>
-   *
-   * <p>Specifies the tenants that may own any entities (e.g. process definition, process instances,
+   * Specifies the tenants that may own any entities (e.g. process definition, process instances,
    * etc.) resulting from this command.
    *
    * <h1>One or more tenants</h1>
    *
    * <p>This can be useful when requesting jobs for multiple tenants at once. Each of the activated
    * jobs will be owned by the tenant that owns the corresponding process instance.
    *
+   * <p>Each time this method is called, the existing tenant IDs are cleared and only the tenant IDs
+   * passed to this method are added.
+   *
```

Comment:
🔧 While this is fine, I'd prefer that users could chain it in any combination `.tenantIds(mydefaults).tenantIds(someOthers).tenantId(anotherSpecificOne)`